src/ape_pm/compiler.py

```diff
@@ -26,18 +26,20 @@ def compile(
         for path in filepaths:
             abi = json.loads(path.read_text())
 
-            source_id = (
-                str(get_relative_path(path, base_path))
-                if base_path and path.is_absolute()
-                else str(path)
-            )
+            if base_path:
+                source_id = str(get_relative_path(path.absolute(), base_path))
+                contract_name = source_id.replace(".json", "").replace("/", ".")
```

Comment:
Was curious if it was important that interface compilers use the "."

Reply:
Not really; I was thinking it was a nice feature and it helps us test that this feature works without relying on `ape-cairo` to do it.

But I don't necessarily need it so perhaps it is best to not change that for now.